index.html

```diff
@@ -20,25 +26,40 @@
     </header>
 
     <section>
-        <img src="Assets/Ellipse 11.png" class="deep_left">
-        <img src="Assets/Rectangle 4.png" class="light_right">
-        <img src="Assets/Ellipse 3.png" class="deep_right">
-        <img src="Assets/Rectangle 6.png" class="light_left">
-      <h1 class="intro">I'm Max <br> Glad to see you</h1>
-      
+      <img src="Assets/Ellipse8.png" class="small_bal" alt="small_ball_1"/>
+      <img src="Assets/Ellipse14.png" class="small_ball" alt="small_ball_2"/>
+      <img src="Assets/Ellipse5.png" class="ball" alt="ball"/>
+      <img src="Assets/Rectangle7.png" class="bar" alt="bar"/>
+      <img src="Assets/Ellipse11.png" class="deep_left" alt="left_ellipse"/>
+      <img src="Assets/Rectangle4.png" class="light_right" alt="right_box"/>
+      <img src="Assets/Ellipse3.png" class="deep_right" alt="right_ellispe"/>
+      <img src="Assets/Rectangle6.png" class="light_left" alt="left_box"/>
+      <h1 class="intro">
+        <strong>
+        I'm Max <br />
+        Glad to see you!
+      </strong>
+      </h1>
+
       <p class="description">
         I'm a software developer! I can help you build a product , feature or
         website. Look through some of my work and experience! If you like what
         you see and have a project you need coded, don't hestiate to contact me.
       </p>
       <span class="connect">LET'S CONNECT</span>
       <div class="social">
-        <span class="fa fa-twitter twitter"></span>
-        <span class="fa fa-linkedin"></span>
-        <span class="fa fa-medium-m"></span>
-        <span class="fa fa-github"></span>
-        <span class="fa-forumbee"></span>
+        <ul>
+          <li><a href="#"><span class="fa fa-twitter twitter"></span></a></li>
+          <li><a href="#"><span class="fa fa-linkedin"></span></a></li>
+          <li><a href="#"><span class="fa fa-medium"></span></a></li>
+          <li><a href="#"><span class="fa fa-github"></span></a></li>
+          <li><a href="#"><span class="fa fa-hand-peace"></span></a></li>
+        </ul>
       </div>
     </section>
+    <img src="./Assets/Ellipse4.png" class="roll_ball" alt="roll_ball" />
+    <img src="./Assets/Rectangle5.png" class="roll_bar" alt="roll_bar"/>
+    <img src="./Assets/Ellipse12.png" class="bowl" alt="bowl"/>
+    <img src="./Assets/Ellipse13.png" class="hook" alt="hook"/>
```

Comment:
- [OPTIONAL] I suggest you select the group of these images from the Figma design and include it as a background and not use the img tag.